app/_components/_popup/base.tsx

```
@@ -0,0 +1,35 @@
+interface IBasePopupProps {
+  children: React.ReactNode;
+  className?: string;
+  height?: 'auto' | 'full' | `${number}px` | `${number}%`;
+  onClose?: () => void;
+  width?: 'auto' | 'full' | `${number}px` | `${number}%`;
+}
+
+export function Wrapper({ children, width = 'auto', height = 'auto', className, onClose }: IBasePopupProps) {
+  return (
+    <div className={className || 'w-full h-full absolute top-0 left-0 z-[100] flex flex-col items-center justify-center'}>
+      <div className="w-full h-full relative bg-black bg-opacity-50" onClick={onClose} />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)